defaults/main.yml

```diff
@@ -8,12 +8,15 @@ haproxy_group: haproxy
 haproxy_frontend_name: 'hafrontend'
 haproxy_frontend_bind_address: '*'
 haproxy_frontend_port: 80
+haproxy_frontend_ssl_port: 443
 haproxy_frontend_mode: 'http'
+haproxy_ssl_pem: ''
```

Comment:
Hi @geerlingguy, 
Wasn't able to do it due to overloading. Will done today. Thanx for your patience.